its/src/test/kotlin/org/sonarlint/intellij/its/tests/SecurityHotspotTabTest.kt

```diff
@@ -0,0 +1,213 @@
+ */
+package org.sonarlint.intellij.its.tests
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.fixtures.ActionButtonFixture.Companion.byTooltipText
+import com.intellij.remoterobot.fixtures.ContainerFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import com.intellij.remoterobot.utils.keyboard
+import com.intellij.remoterobot.utils.waitFor
+import com.sonar.orchestrator.Orchestrator
+import com.sonar.orchestrator.build.MavenBuild
+import com.sonar.orchestrator.container.Edition
+import com.sonar.orchestrator.container.Server
+import com.sonar.orchestrator.locator.FileLocation
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.condition.DisabledIf
+import org.sonarlint.intellij.its.BaseUiTest
+import org.sonarlint.intellij.its.fixtures.clickWhenEnabled
+import org.sonarlint.intellij.its.fixtures.dialog
+import org.sonarlint.intellij.its.fixtures.idea
+import org.sonarlint.intellij.its.fixtures.jPasswordField
+import org.sonarlint.intellij.its.fixtures.jRadioButtons
+import org.sonarlint.intellij.its.fixtures.jbTextField
+import org.sonarlint.intellij.its.fixtures.jbTextFields
+import org.sonarlint.intellij.its.fixtures.tool.window.toolWindow
+import org.sonarlint.intellij.its.utils.OrchestratorUtils
+import org.sonarqube.ws.client.HttpConnector
+import org.sonarqube.ws.client.WsClient
+import org.sonarqube.ws.client.WsClientFactories
+import org.sonarqube.ws.client.users.CreateRequest
+import org.sonarqube.ws.client.usertokens.GenerateRequest
+import java.io.File
+import java.time.Duration
+
+
+const val SECURITY_HOTSPOT_PROJECT_KEY = "sample-java-hotspot"
+
+@DisabledIf("isCLion", disabledReason = "No security hotspots in CLion")
+class SecurityHotspotTabTest : BaseUiTest() {
+
+    @Test
+    fun should_request_the_user_to_bind_project_when_not_bound() = uiTest {
+        openExistingProject("sample-java-hotspot", true)
+        verifySecurityHotspotTabContainsMessages(this, "The project is not bound to SonarQube 9.7+")
+    }
+
+    @Test
+    fun should_display_security_hotspots() = uiTest {
+        openExistingProject("sample-java-hotspot", true)
+        bindProjectFromPanel()
+
+        openFile("src/main/java/foo/Foo.java", "Foo.java")
+
+        verifySecurityHotspotTreeContainsMessages(this, "Make sure using this hardcoded IP address is safe here.")
+    }
+
+    private fun bindProjectFromPanel() {
+        with(remoteRobot) {
+            idea {
+                toolWindow("SonarLint") {
+                    ensureOpen()
+                    tab("Security Hotspots") { select() }
+                    content("SecurityHotspotsPanel") {
+                        jLabel("Configure binding").click()
+                    }
+                }
+                dialog("Project Settings") {
```

Comment:
I think this part and below already exists in other tests, do you think we could factorize ?